proto/rpc_delete_book.proto

```
@@ -0,0 +1,9 @@
+syntax = "proto3";
+
+package pb;
```

Buf (1.47.2)

Files with package "pb" must be within a directory "pb" relative to root but were in directory "proto".

(PACKAGE_DIRECTORY_MATCH)